js/components/ui/Image/Image.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Компонент для создания изображения
+ * @function Image
+ * @typedef {object} Image - контент для изображения
+ * @property {string} source
+ * @property {string} description
+ * @param {Image} image
+ * @param {string} className
+ * @returns {string} html or empty
+ */
```

Comment:
Split into two JSDocs
The first JSDoc to declare the type
The second JSDoc for the function description